components/patient-data/rest/src/main/java/org/phenotips/data/rest/PatientsFetchResource.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see http://www.gnu.org/licenses/
+ */
+package org.phenotips.data.rest;
+
+import org.phenotips.rest.ParentResource;
+import org.phenotips.rest.Relation;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+/**
+ * Resource for working with multiple patient records, identified by their given "external" identifiers.
+ *
+ * @version $Id$
+ * @since 1.3M5
+ */
+@Path("/patients/fetch")
+@Relation("https://phenotips.org/rel/patientRecord")
```

Comment:
This isn't necessarily one patient record, so I would call it `"https://phenotips.org/rel/patientRecords"` (plural).